bodhi/tests/server/functional/test_admin.py

```diff
@@ -0,0 +1,44 @@
+# This program is free software; you can redistribute it and/or
```

Comment:
Above this line, let's add:

```
# Copyright 2017 Red Hat, Inc.
#
# This file is part of Bodhi.
#
```

I've been meaning to add these declarations to the existing files, but I'll need to do some research to find out what years to claim on each file so it's not super easy.